ddtrace/contrib/tornado/stack_context.py

```diff
@@ -31,7 +31,7 @@ class TracerStackContext(DefaultContextProvider):
         """
 
         def __init__(self):
-            self._active = True
+            super(TracerStackContext, self).__init__()
```

Comment:
Ah yeah `_DD_CONTEXTVAR.set(None)` would be wrong for the tornado manager.

The tornado context management is a bit of a beast of its own. There isn't a single context provider that gets used (there is just one that the tracer has) but the tornado management will create new providers using this "stack context" API provided by tornado for every new span that is created.

This is super 🤯 to reason about and I really don't like it... fortunately newer versions of tornado are asyncio based rather than twisted based....

Reply:
Ok, though that's a real problem. You can't inherit from a class and not call `super()`. I mean you can, but it's a pretty bad practice and this hitting me right now.

Is there a clean way to make `_DD_CONTEXTVAR.set(None)` right or should I just hack around it? 😢

Reply:
@Kyle-Verhoog thoughts?

Reply:
`_DD_CONTEXTVAR.set(None)` would only make sense if there was a backport of `contextvars` in tornado but then the whole stack context thing wouldn't be required if we had that 😕 

Given that this is only for old tornado I think the best option is to hack around it @jd

Reply:
Alright I removed `super`, I'll hack around in the next PR!